tests/fault_injector/conftest.py

```diff
@@ -1,6 +1,7 @@
 import pytest
 from traci import vehicle
 
+from src.event_bus.event_bus import EventBus
 from src.implementor.models import Run, SimulationConfiguration, Token
 from src.interlocking_component.route_controller import IInterlockingDisruptor
 from src.logger.logger import Logger
```

Comment:
How come you still need the Logger?

Reply:
Prior to this refactoring tests using fixtures from this file used the `Logger` and expected `LogEntry`s to be created. Now `Components` have the `EventBus` instead of the `Logger`. Nonetheless `LogEntry`s still need to be created. Therefore the Logger still needs to be instantiated here.